Notebooks/DC2 Coadd Run1.1p GCR access -- Tutorial (part1).ipynb

```diff
@@ -0,0 +1,1401 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DC2 Coadd Run1.1p GCR access\n",
+    "\n",
+    "__Purpose__: This tutorial will illustrate the basics of accessing the merged coadd catalogs through the Generic Catalog Reader (GCR, https://github.com/yymao/generic-catalog-reader) as well as how to select useful samples of stars/galaxies from the DM outputs.\n",
+    "\n",
+    "\n",
+    "Note: This notebook is intended to be run through the JupyterHub NERSC interface available here: https://jupyter-dev.nersc.gov. To setup your NERSC environment, please follow the instructions available here: https://confluence.slac.stanford.edu/display/LSSTDESC/Using+Jupyter-dev+at+NERSC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pylab inline"
```

Comment:
I'm very much a fan of being explicitly about `pyplot` and `numpy` use.  These tutorials also will end up being examples that people will copy for style.

I would suggest changing this one line to:
```
%matplotlib inline

import matplotlib.pyplot as plt
import numpy as np
```

and then encourage use of `np.` and `plt.` as neeeed below.